package.json

```diff
@@ -161,12 +161,42 @@
           "title": "Use asciidoctor-pdf",
           "description": "[asciidoctor-pdf](https://github.com/asciidoctor/asciidoctor-pdf) must be installed in your system.",
           "type": "boolean",
-          "default": false
+          "default": false,
+          "order": 1
         },
         "openWithExternal": {
           "title": "Automatically open the PDF after generation with the default PDF viewer.",
           "type": "boolean",
-          "default": true
+          "default": true,
+          "order": 2
+        },
+        "pdfStyle": {
+          "title": "Style (-a pdf-style=)",
+          "description": "The name of the YAML theme file to load. <br> https://github.com/asciidoctor/asciidoctor-pdf/blob/master/docs/theming-guide.adoc#applying-your-theme",
+          "type": "string",
+          "default": "",
+          "order": 3
+        },
+        "pdfStylesDir": {
+          "title": "Fonts directory (-a pdf-stylesdir=)",
+          "description": "The directory where the theme file is located. _Specifying an absolute path is recommended._ <br> https://github.com/asciidoctor/asciidoctor-pdf/blob/master/docs/theming-guide.adoc#applying-your-theme",
+          "type": "string",
+          "default": "",
+          "order": 4
+        },
+        "pdfFontsDir": {
+          "title": "Fonts directory (-a pdf-fontsdir=)",
+          "description": "The directory where the fonts used by your theme, if any, are located. _Specifying an absolute path is recommended._ <br> https://github.com/asciidoctor/asciidoctor-pdf/blob/master/docs/theming-guide.adoc#applying-your-theme",
+          "type": "string",
+          "default": "",
+          "order": 5
+        },
+        "arguments": {
+          "title": "Command line additional arguments",
+          "description": "ex: `-a pdf-stylesdir=~/resources/themes -a pdf-style=basic`",
```

Comment:
Now that we have dedicated options for pdf-stylesdir and pdf-style, the example here should be something like "-a pdf-page-size=Letter".